src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -146,11 +146,17 @@ const EditShiftHours = ({
     );
 
     const endTime = formData[`${inputName}-finish-time`] || null;
+    const endDate = `${formData['finishDate-year']}-${formData['finishDate-month']}-${formData['finishDate-day']}`;
```

Comment:
Could we give this variable a more descriptive name? Something like `modifiedEndDate` to differentiate it from our other end dates

Reply:
This variable has now been removed